frontend/src/services/jokes.js

```diff
@@ -0,0 +1,90 @@
+const getPage = async (numJokesPerPage, page) => {
+    const response = await fetch(`/joke/?num_jokes_per_page=${numJokesPerPage}&page=${page}`);
+
+    if (!response.ok) {
+        const message = `An error has occured: ${response.status}`;
+        throw new Error(message);
+    }
+
+    const body = await response.json();
+    console.log(body)
+    return body;
```

Comment:
```suggestion
    return await response.json();
```

Was the console logging for debug?